src/Compilers/VisualBasic/Portable/BoundTree/Expression.vb

```diff
@@ -206,6 +214,22 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
             Return OperationKind.AssignmentExpression
         End Function
+
+        Public Overrides Sub Accept(visitor As IOperationVisitor)
+            If Me.ExpressionKind() = OperationKind.CompoundAssignmentExpression Then
+                visitor.VisitCompoundAssignmentExpression(Me)
+            Else
+                visitor.VisitAssignmentExpression(Me)
+            End If
+        End Sub
```

Comment:
For all those types implement multiple IOperation interfaces, I will talk to @JohnHamby and see if he has any opinions. I will create another PR for that if we ended up changing it.